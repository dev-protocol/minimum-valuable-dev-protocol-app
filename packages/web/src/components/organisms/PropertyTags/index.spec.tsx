import React from 'react'
import { render } from '@testing-library/react'
import { PropertyTags } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/fixtures/dev-for-apps/hooks')

describe(`${PropertyTags.name}`, () => {
  test('Snapshot', () => {
    const component = render(<PropertyTags />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
