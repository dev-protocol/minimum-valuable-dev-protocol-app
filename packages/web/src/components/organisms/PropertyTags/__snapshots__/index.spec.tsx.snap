// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PropertyTags Snapshot 1`] = `
<body>
  <div>
    <div
      class="uh7l8d-0"
    >
      <p>
        Property Tags
      </p>
    </div>
  </div>
</body>
`;
